src/driver/aurora-data-api/AuroraDataApiDriver.ts

```diff
@@ -583,6 +583,40 @@ export class AuroraDataApiDriver implements Driver {
         }
     }
 
+    /**
+     * Normalizes "setDefault" value of the column.
+     */
+     normalizeSetDefault(columnMetadata: ColumnMetadata): string | undefined {
+      const defaultValue = columnMetadata.setDefault;
+
+      if (defaultValue === null) {
```

Comment:
Can you explain more here? I was under the impression === was the more type safe option.

Reply:
Also this function is basically a duplicate of the pre-existing normalizeDefault function. The only reason I duplicated instead of merged was to remove support of using strings to run database commands via the setDefault method. Also likely wont get to this line as this function should not be called in the event column.setDefault was null anyway.

Reply:
I suggested this change becuase there might be `undefined` in defaultValue.

Reply:
kill me but I don't see a difference with the `normalizeDefault` function? Can you point me please to a difference?

Reply:
This PR's code snippet is what I've seen and don't know where/how `normalizeDefault` is used. I suggested this because of the possible edge case. (English isn't my first language so tell me if I've understand wrong what you're saying)

Reply:
If this change isn't necessary you can resolve this conversation

Reply:
They are essentially the same function, normalizeDefault definition is right above this line in all the driver files. It just looks at the columnMetadata.setDefault option I have added in this PR instead of the pre-existing columnMetadata.default, and also the normalizeSetDefault version of the function removes any normalizing for default DB funcitons like CURRENT_TIMESTAMP and such.